src/Appwrite/Specification/Format/Swagger2.php

```diff
@@ -342,6 +342,8 @@ public function parse(): array
                     case 'Utopia\Validator\ArrayList':
                     case 'Appwrite\Utopia\Database\Validator\Queries\Buckets':
                     case 'Appwrite\Utopia\Database\Validator\Queries\Collections':
+                    case 'Appwrite\Utopia\Database\Validator\Queries\Indexes':
+                    case 'Appwrite\Utopia\Database\Validator\Queries\Attributes':
```

Comment:
We will also need to add these cases in `src/Appwrite/GraphQL/Types/Mapper.php`

Reply:
done this